run-cnf-suites.sh

```diff
@@ -162,4 +162,7 @@ cd ./cnf-certification-test || exit 1
 	${FOCUS_STRING} \
 	${SKIP_STRING} \
 	"${LABEL_STRING}" \
-	${GINKGO_ARGS}
+	${GINKGO_ARGS} |& tee $OUTPUT_LOC/tnf-execution.log
+
+# preserving the exit status
+exit ${PIPESTATUS[0]}
```

Comment:
`set -o pipefail` is a more general solution, because it fails if any command fails, `tee` could fail on permission, full disk, etc.